src/index.spec.js

```diff
@@ -14,7 +14,6 @@ describe('babel plugin', () => {
   //       to starting subprocesses.
   var procPool
   beforeEach(() => { procPool = new ProcessPool })
-  afterEach(() => { procPool.destroy() })
```

Comment:
For more than one test, if this line is present, the tests won't pass. Removing it doesn't seem to harm anything.